src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -233,30 +233,43 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
                         filesToSend.Add(viewModel.DumpRecordedCommands());
                     }
 
-                    var extras = string.Join(" ", filesToSend.Select(f => "/EXTRA \"" + f + "\""));
-
                     string appConfig = "";
                     if (model != null)
                     {
                         var appName = GetHostAppName(model);
-                        appConfig = $@"<ProductInformation name=\""{appName}\"" build_version=\""{model.Version}\"" " +
-                                    $@"registry_version=\""{model.Version}\"" registry_localeID=\""{CultureInfo.CurrentCulture.LCID}\"" uptime=\""0\"" " +
-                                    $@"session_start_count=\""0\"" session_clean_close_count=\""0\"" current_session_length=\""0\"" />";
+                        appConfig = $"<ProductInformation name=\"{appName}\" build_version=\"{model.Version}\" " +
+                                    $"registry_version=\"{model.Version}\" registry_localeID=\"{CultureInfo.CurrentCulture.LCID}\" uptime=\"0\" " +
+                                    $"session_start_count=\"0\" session_clean_close_count=\"0\" current_session_length=\"0\" />";
                     }
 
-                    string dynConfig = string.Empty;
                     string dynName = viewModel?.Model.CurrentWorkspace.Name;
-                    if (!string.IsNullOrEmpty(dynName))
-                    {
-                        dynConfig = $"/DWG {dynName}";
-                    }
 
                     var miniDumpFilePath = CreateMiniDumpFile(cerDir.FullName);
                     var upiConfigFilePath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "upiconfig.xml");
 
-                    var cerArgs = $"/UPITOKEN \"{upiConfigFilePath}\" /DMP \"{miniDumpFilePath}\" /APPXML \"{appConfig}\" {dynConfig} {extras} /USEEXCEPTIONTRACE";
+                    using (var cerDLL = new CerDLL(cerToolPath)) 
+                    {
+                        cerDLL.ToggleCER(true);
+                        cerDLL.RegisterUPI(upiConfigFilePath);
+
+                        if (!string.IsNullOrEmpty(dynName))
+                        {
+                            cerDLL.SetStringParam(ReportStringParamKey.StringKeyDwg, dynName);
+                        }
+
+                        foreach (var file in filesToSend)
+                        {
+                            cerDLL.SetMultiStringParam(ReportMultiStringParamKey.MultiStringKeyExtraFile, file, filesToSend.Count + 1);
```

Comment:
will `SetMultiStringParam` actually send the file content ?

Reply:
why filesToSend.Count + 1 ?

Reply:
I can maybe remove +1. Let me check.